api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -244,6 +244,9 @@ message ResponseFlags {
 
   // Indicates there was an HTTP protocol error on the downstream request.
   bool downstream_protocol_error = 19;
+
+  // Indicates there was a max stream duration reached on the upstream request.
+  bool upstream_max_stream_duration_reached = 20;
```

Comment:
@htuch this is an interesting API question. For data protos, what do we do about v2? I'm actually questioning whether data protos should be versioned in the same way. See also the validation changes for the new data flag. cc @envoyproxy/api-shepherds

Reply:
Ping on @htuch to advise on this one. I think we need to have a discussion about how to handle data proto changes.

Reply:
As things stand, everything is major versioned in the same way. If we want to make exceptions, I suggest a PR to update https://github.com/envoyproxy/envoy/blob/master/api/API_VERSIONING.md